tests/feature_toggle/test_dialup.py

```diff
@@ -4,19 +4,17 @@
 from samtranslator.feature_toggle.dialup import *
 
 
-class TestBaseDialup(TestCase):
-    def test___str__(self):
-        region_config = {}
-        dialup = BaseDialup(region_config)
-        self.assertEqual(str(dialup), "BaseDialup")
```

Comment:
note: this test has been removed since it makes no sense to create a `BaseDialup()` on its own. It is now an abstract class